lib/index.js

```diff
@@ -34,61 +29,21 @@ module.exports = class SpikeDatoCMS {
     })
 
     compiler.plugin('emit', (compilation, done) => {
-      if (this.json) {
-        writeJson(compilation, this.json, this.addDataTo.dato)
-      }
```

Comment:
that was just temporary, I can re-enable it, but I thought that was redundant, as you can use the the `singlePages` option to do the same. Well, not at the moment, because the `writeTemplate` is not using webpack, right?